src/utilities/effects.ts

```diff
@@ -5003,6 +5657,34 @@ const effects = {
     }
   },
 
+  async updateDerivationGroupAcknowledged(
+    plan: Plan | undefined,
+    derivation_group_name: string,
+    updatedAt = new Date(),
```

Comment:
I don't think that this needs to be a parameter. Theoretically, this should always be UTC now.